config/locales/es-MX.yml

```diff
@@ -286,6 +286,55 @@ es-MX:
       solidus:
         long: '%B %d, %Y %-l:%M %p'
         short: "%b %-d '%y %-l:%M%P"
+  date:
+    abbr_day_names:
+      - lun
+      - mar
+      - mie
+      - jue
+      - vie
+      - sab
+      - dom
+    abbr_month_names:
+      -
+      - ene
+      - feb
+      - mar
+      - abr
+      - may
+      - jun
+      - jul
+      - ago
+      - sep
+      - oct
+      - nov
+      - dic
+    day_names:
+      - lunes
+      - martes
+      - miércoles
+      - jueves
+      - viernes
+      - sábado
+      - domingo
+    formats:
+      default: "%d/%m/%Y"
+      short: "%e %b"
+      long: "%e %B %Y"
+    month_names:
+      -
+      - enero
+      - febrero
+      - marzo
+      - abril
+      - mayo
+      - junio
+      - julio
+      - agosto
+      - septiembre
+      - octubre
+      - noviembre
+      - diciembre
```

Comment:
Shouldn’t all these names be capitalized?

Reply:
I was following the same format in other locales, but I made the change, thanks